routes/orders.js

```diff
@@ -3,10 +3,10 @@ const { getAllOrders, getCartByUser, createOrder } = require('../db/orders');
 const { getOrderProductById, addProductToOrder, updateOrderProduct } = require('../db/order_products');
 const { requireUser, requireAdmin } = require('./utils');
 
-ordersRouter.get('/', requireAdmin, async (req, res, next) => {
```

Comment:
Why did you remove the requireAdmin middleware?

Reply:
That was actually an oversight! I need to add that back in, good catch. I think I had taken it out for testing purposes and just neglected to put it back.